lib/datadog/statsd/telemetry.rb

```diff
@@ -11,10 +11,11 @@ class Telemetry
       attr_reader :bytes_dropped
       attr_reader :packets_sent
       attr_reader :packets_dropped
-      attr_reader :estimate_max_size
 
-      def initialize(disabled, flush_interval, global_tags: [], transport_type: :udp)
-        @disabled = disabled
+      # Rough estimation of maximum telemetry message size without tags
+      MAX_TELEMETRY_MESSAGE_SIZE_WT_TAGS = 50 # bytes
```

Comment:
Yup, that's the longest message possible from the telemetry (packets_dropped) with a 7 digit number. If the person adds tags, it would also increase the estimation of the maximum possible line.